### PR TITLE
Adding documentation

### LINKED ISSUE #3: Plugin documentation
`README.md` should be modified to contain more information about the plugin. For example, all commands with descriptions and all permissions.

### PR DESCRIPTION
**Pull request by EmiCB**
Added JavaDoc and some inline comments to all files

### CODE REVIEW

File: src/main/java/edu/whimc/positiontracker/sql/Queryer.java [hunk 1]
```diff
@@ -76,11 +105,16 @@ public Queryer(Tracker plugin, Consumer<Boolean> callback) {
...
+    /**
+     * Stores the position data.
+     */
     public void storePositionData() {
+        // get position data for all online players
         final List<PositionEntry> entries = Bukkit.getOnlinePlayers().stream()
                 .map(PositionEntry::new)
                 .collect(Collectors.toList());
 
+        // add data to database if connection exists
```
Comment (jackah2) [inline src/main/java/edu/whimc/positiontracker/sql/Queryer.java]:
```suggestion
        // add data to database asynchronously
```
This is a [try-with statement](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) which automatically closes the connection after it's done. Technically the connection doesn't exist at this point.

File: src/main/java/edu/whimc/positiontracker/sql/Queryer.java [hunk 2]
```diff
@@ -56,14 +76,23 @@ public void addInsertionToBatch(PreparedStatement statement) throws SQLException
 
     }
 
+    /** The formatting for a data insertion. */
```
Comment (jackah2) [inline src/main/java/edu/whimc/positiontracker/sql/Queryer.java]:
```suggestion
    /** SQL query to insert position data. */
```

File: src/main/java/edu/whimc/positiontracker/sql/Queryer.java [hunk 3]
```diff
@@ -42,6 +56,12 @@ public PositionEntry(Player player) {
             this.time = new Timestamp(System.currentTimeMillis());
         }
 
+        /**
+         * Adds an SQL Insertion with the PositionEntry data.
```
Comment (jackah2) [inline src/main/java/edu/whimc/positiontracker/sql/Queryer.java]:
```suggestion
         * Adds this entry's data to the PreparedStatement's batch.
```

**Review by jackah2 — CHANGES_REQUESTED** [review]
A few minor changes. Everything else looks good :)

**Comment by EmiCB** [comment]
Oops, I also forgot to update the readme!

**Review by jackah2 — APPROVED** [review]
Looks good!

**Comment by jackah2** [comment]
Resolves #3

**Pull request merged**